doc/source/whatsnew/v1.0.1.rst

```diff
@@ -70,6 +70,7 @@ Indexing
 
 -
 -
+- Bug where assigning to a ``Series`` using a IntegerArray / BooleanArray as a mask would raise ``TypeError`` (:issue:`31446`)
```

Comment:
:class:`Series` instead of ``Series`` so that there's a link in the docs?

Reply:
yeah, tbh i wasn't quite sure about the rule here, since i see three different representations used in whatsnew
```
:class:`Series`, ``Series``, Series
```
but seems having :class:`Series` is more consistent, will change